packages/dds/tree/src/test/utils.ts

```diff
@@ -1300,3 +1302,23 @@ export function moveWithin(
 ) {
 	editor.move(field, sourceIndex, count, field, destIndex);
 }
+
+/**
+ * Clones a batch of revertibles for a target branch.
+ * @param revertibles - Array of revertibles to clone
+ * @param targetBranch - The target branch to clone the revertibles for
+ * @returns Array of cloned revertibles, maintaining the same order as the input
+ */
+export function cloneRevertiblesInBatch(
```

Comment:
nit: I think just `cloneRevertibles` is sufficient for a name, you can see from the type you pass in an array. The 'batch' wording is typically used in code that is doing asynchronous processing of more than 1 item.

Reply:
That's also the name that I had in mind initially. I wasn't sure if the plural (_Revertibles_) will be straightforward to the users to imply that it supports cloning in group. Glad that you pointed this out, will change the name to `cloneRevertibles`.